.gitignore

```diff
@@ -1,4 +1,7 @@
+build*
 buildroot*
+ccache
 download
 grubd
 output
+src/etc/cirros/version
```

Comment:
IMHO it is bad style to ignore a file that is actually present in the git tree. Can we drop it there and resort to always generating it in the build process?

Reply:
Added commit which drops it.